lib/assessment.rb

```diff
@@ -14,6 +15,7 @@ def longest(enum)
   # Assume each entry implements a `+` method
   def sum(enum)
     # Your code goes here
+    enum.reduce(0) { |memo, item| memo + item}
```

Comment:
Correct. This could have been written even more shortly if you used the `&` trick:
`enum.reduce(&:+)`